extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.ViewPageTarget.js

```diff
@@ -32,45 +32,90 @@ OO.inheritClass( ve.init.wikia.ViewPageTarget, ve.init.mw.ViewPageTarget );
 
 /* Static Properties */
 
-ve.init.wikia.ViewPageTarget.static.toolbarGroups = [
-	// History
-	{ include: [ 'undo' ] },
-	// Format
-	{
-		type: 'menu',
-		indicator: 'down',
-		title: OO.ui.deferMsg( 'visualeditor-toolbar-format-tooltip' ),
-		include: [ { group: 'format' } ],
-		promote: [ 'paragraph' ],
-		demote: [ 'preformatted' ],
-		exclude: [ 'heading1' ]
-	},
-	// Style
-	{ include: [ 'bold', 'italic', 'link' ] },
-	{
-		type: 'list',
-		icon: 'text-style',
-		indicator: 'down',
-		title: OO.ui.deferMsg( 'visualeditor-toolbar-style-tooltip' ),
-		include: [ 'subscript', 'superscript', 'strikethrough', 'underline', 'indent', 'outdent', 'clear' ]
-	},
-	// Insert
-	{
-		type: 'list',
-		label: OO.ui.deferMsg( 'visualeditor-toolbar-insert' ),
-		indicator: 'down',
-		include: [ 'wikiaMediaInsert', 'wikiaSingleMedia', 'wikiaInfoboxInsert', 'number', 'bullet', 'wikiaMapInsert', 'wikiaTemplateInsert', 'reference', 'referencesList', 'insertTable' ]
-	},
-	// Table
-	{
-		header: OO.ui.deferMsg( 'visualeditor-toolbar-table' ),
-		type: 'list',
-		icon: 'table-insert',
-		indicator: 'down',
-		include: [ { group: 'table' } ],
-		demote: [ 'deleteTable' ]
-	}
-];
+if (ve.init.wikia.ToolbarABTestVariantNumber() === 1) {
```

Comment:
Instead of having two very similar arrays I suggest considering defining array once and then modifying it slightly - in case of the experiment (that's what we've done before for similar experiment).